projects/sport-data-valley/seeds/questionnaires/mmm_questionnaire.rb

```diff
@@ -0,0 +1,508 @@
+              en: "It takes a long time for me to recover from a stressful experience." },
+    options: agreement_options,
+    show_otherwise: false
+  },
+  {
+    id: 'getting_over_setbacks',
+    type: :likert,
+    required: true,
+    section_end: true,
+    title: {  nl: "Het kost me meestal veel tijd om over tegenslagen in mijn leven heen te komen",
+              en: "It usually takes a long time for me to get over setbacks in my life." },
+    options: agreement_options,
+    show_otherwise: false
+  },
+  {
+    id: 'study_participation_experience',
+    type: :textarea,
+    required: true,
+    section_start: {
+      nl: "Open vragen",
+      en: "Open questions"
+    },
+    title: {  nl: "Hoe ervaar je momenteel je deelname aan dit onderzoek?",
+              en: "How do you currently experience your participation in this study?" }
+  },
+  {
+    id: 'does_questionnaire_affect',
+    type: :radio,
+    required: true,
+    title: {  nl: "Heeft het invullen van de vragenlijsten invloed op wat je doet en/of hoe je je voelt?",
+              en: "Does completing the questionnaires affect what you do and/or how you feel?" },
+    options: [
+      {
+        title: {  nl: "Ja",
+                en: "Yes" },
+        value: "yes",
+        shows_questions: ['how_questionnaire_affects']
+      },
+      {
+        title: {  nl: "Nee",
+                en: "No" },
+        value: "no"
+      }
+    ],
+    show_otherwise: false
+  },
+  {
+    id: 'how_questionnaire_affects',
+    type: :textarea,
+    hidden: true,
+    required: true,
+    title: {  nl: "Zo ja, geef dan aan wat voor invloed het op jou heeft:",
+              en: "If so, please indicate how it affects you:" },
+  },
+  {
+    id: 'does_tracker_affect',
+    type: :radio,
+    required: false,
+    title: {  nl: "Heeft het dragen van de activity tracker invloed op wat je doet en/of hoe je je voelt?",
+              en: "Does wearing the activity tracker affect what you do and/or how you feel?" },
+    options: [
+      {
+        title: {  nl: "Ja",
+                en: "Yes" },
+        value: "yes",
+        shows_questions: ['how_tracker_affects']
+      },
+      {
+        title: {  nl: "Nee",
+                en: "No" },
+        value: "no"
+      }
+    ],
+    show_otherwise: false
+  },
+  {
+    id: 'how_tracker_affects',
+    type: :textarea,
+    required: true,
+    hidden: true,
+    title: {  nl: "Zo ja, geef dan aan wat voor invloed het op jou heeft:",
+              en: "If so, please indicate how it affects you:" },
+  },
+  {
+    id: 'last_remarks',
+    type: :textarea,
+    section_end: true,
+    title: {  nl: "Is er nog iets anders dat je kwijt wil? (Zo niet, dan kan je dit antwoordveld leeg laten)",
+              en: "Is there anything else you would like to share? (If not, you can leave this answer field empty)" },
+  }
+]
+
+questionnaire.content = { questions: question_content, scores: [] }
```

Comment:
Btw, if you need to calculate some scores for the questionnaire, like taking the sum or average of some questions, you can specify them in the `scores` array here. The results are calculated immediately when the questionnaire is completed, and stored in the same hash as the other questionnaire answers.

You can check `psychological_competencies.rb` for an example of a questionnaire with scores.